frontend/model/contracts/voting/proposals.js

```diff
@@ -59,16 +59,20 @@ function voteAgainst (state, { meta, data, contractID }) {
   sbp('okTurtles.events/emit', PROPOSAL_RESULT, state, VOTE_AGAINST, data)
 }
 
+// NOTE: The code is ready to receive different proposals settings,
+// However, we decided to make all settings the same, to simplify the UI/UX proptotype.
+const proposalDefaults = {
+  rule: RULE_PERCENTAGE,
+  expires_ms: 7 * DAYS_MILLIS,
```

Comment:
I noticed that the proposals had different expire days.
- Add/Remove Member proposals expire in 14 days. 
- Group Setting and Proposal Setting expire in 7 days.

Any reason for that? Maybe we could also make it all the same to simplify.

cc @mmbotelho